student.rb

```diff
@@ -1,16 +1,19 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :name, :age
+
```

Comment:
- [ ] Please add an `attr_reader` for `classroom` as suggested by the previous reviewer. We already specified a custom setter function for classroom, we just need to specify a getter to read the classroom data. 😎️

```
attr_reader :classroom
```